src/app/header/titles.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnDestroy, OnInit, Input } from "@angular/core";
+import { TitlesService } from "./titles.service";
+import { Breadcrumb } from "./breadcrumb";
+import { Subscription } from "rxjs";
+import { LoggerFactory, LogLevel, Logger } from "diagnostics/logger";
+
+@Component({
+    selector: `titles`,
+    template: `
+<div class="titles">
+    <ul class="breadcrumbs sme-focus-zone">
```

Comment:
Please remove sme-focus-zone, which is not necessary anymore.

Reply:
fixed